### PR TITLE
[MJLINK-49] documentation update

### PR DESCRIPTION
**Pull request by bmarwell**
- remove alpha release text.
  - Do not tell the user they need a multi-module project.
  - remove superfluous '}'.

----

Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [X] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MJLINK) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [X] Each commit in the pull request should have a meaningful subject line and body.
 - [X] Format the pull request title like `[MJLINK-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MJLINK-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [X] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [X] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [X] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [X] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [X] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: src/site/apt/usage.apt.vm [hunk 1]
```diff
@@ -76,11 +76,15 @@ Usage
...
-  In other words it is not possible to create a Run Time Image from a single Maven
-  Project within the same single Maven Project.
-  Let us assume you have a multi module structure which contains two modules <<mod-1>> and <<mod-2>> 
+  You can use the maven-jlink-plugin the same way you use the maven-jar-plugin, maven-war-plugin,
+  maven-assembly-plugin or any other artifact-generating plugin.
+  You can choose to have a separate module for the artifact generation by specifying
+  <<< <packaging>jlink</packaging> >>>.
+
+  You can also add it to your existing project by adding an explicit execution, probably
+  in the <<<package>>> phase.
```
[the remark this answers is not in the document]
Comment (bdemers) [reply src/site/apt/usage.apt.vm:85]: remove "probably" to make it sound more confident

File: src/site/apt/usage.apt.vm [hunk 2]
```diff
@@ -76,11 +76,15 @@ Usage
...
+
+  Let us assume you have a multi module structure which contains two modules <<mod-1>> and <<mod-2>>
```
Comment (bdemers) [inline src/site/apt/usage.apt.vm:87]:
```suggestion
  Let's assume you have a multi module structure that contains two modules <<mod-1>> and <<mod-2>>
```

File: src/site/apt/usage.apt.vm (hunk 1 above)
[the remark this answers is not in the document]
Comment (bdemers) [reply src/site/apt/usage.apt.vm:85]: nit: multiple lines start with the same "you can", consider using a bulleted list to avoid the repetition

**Review by bdemers — APPROVED** [review]

**Comment by leif81** [comment]
Lgtm!

**Comment by bmarwell** [comment]
Jenkins build green: https://ci-builds.apache.org/blue/organizations/jenkins/Maven%2Fmaven-box%2Fmaven-jlink-plugin/detail/MJLINK-49_doc/1/pipeline/

**Pull request merged**